### ISSUE TITLE
HEIP  dreambooth Error when exporting smpler image None of the inputs have requires_grad=True. Gradients will be None

### ISSUE THREAD

**Issue by ja1496** [kohya-ss/sd-scripts#1036]
What should I do? And I was reminded  ：
 }
 is outdated. `steps_offset` should be set to 1 instead of 0. Please make sure to update the config accordingly as leaving `steps_offset` might led to incorrect results in future versions. If you have downloaded this checkpoint from the Hugging Face Hub, it would be very nice if you could open a Pull request for the `scheduler/scheduler_config.json` file
  deprecate("steps_offset!=1", "1.0.0", deprecation_message, standard_warn=False)
 
C:\Users\.......（Inconvenient to display）

warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
